src/lammpsio/box.py

```diff
@@ -65,36 +102,23 @@ def cast(cls, value):
 
     @classmethod
     def from_matrix(cls, low, matrix, force_triclinic=False):
-        """Create a Box from low and matrix.
+        """Cast from an origin and matrix.
 
         Parameters
         ----------
         low : list
             Origin of the box.
-        matrix : :class:`numpy.ndarray`
-            Upper triangular matrix in LAMMPS style::
-
-                [[lx, xy, xz],
-                 [0, ly, yz],
-                 [0, 0, lz]]
+        matrix : `numpy.ndarray`
+            Box matrix.
         force_triclinic : bool
             If ``True``, forces the box to be triclinic even if the tilt
             factors are zero. Default is ``False``.
 
         Returns
         -------
-        :class:`Box`
+        `Box`
             A simulation box.
 
-        Raises
```

Comment:
I don't really like the raises sections because they're too hard to keep up to date. Please remove throughout if I missed any.